app/views/cms/content_nodes/_form.html.erb

```diff
@@ -46,6 +46,31 @@
             <%= f.input :access do %>
               <%= f.select :access, [[t('public'), 'public'], [t('private'), 'private']] %>
             <% end %>
+            <div class="row">
+              <div class="small-2 columns">
+                <label class="right inline">Kunden-content</label>
+              </div>
+              <div class="small-10 columns">
+                <div class= "row">
+                  <% RabsyGroup.all.order(:name).each_slice((RabsyGroup.count.to_f / 3).ceil) do |slice| %>
```

Comment:
I think switching `all` and `order` is better as sorting will be done in the db.

Reply:
It does `SELECT ... ORDER BY ...` either way. But `.all` can be completely removed from here and that's what I will do